test/unit_tests/unittest_start_request.cpp

```diff
@@ -35,7 +35,7 @@ ::testing::AssertionResult
 DecodingEquals(StartRequest::RawType const& data, std::size_t offset, T expected, const Endian& endian = Endian::LITTLE)
 {
   T actual_val;
-  memcpy(&actual_val, (char*)&data + offset, sizeof(T));
+  memcpy(&actual_val, &data.front() + offset, sizeof(T));
```

Comment:
```suggestion
  memcpy(&actual_val, &data.data() + offset, sizeof(T));
```
https://stackoverflow.com/a/10740266/1493204

Reply:
Done